arcgis-ios-sdk-samples/Display information/Display grid/DisplayGrid.storyboard

```diff
@@ -1,12 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The console is displaying this warning when the popover is presented:

```[Warning] Warning once only: Detected a case where constraints ambiguously suggest a height of zero for a tableview cell's content view. We're considering the collapse unintentional and using standard height instead.```

That is because none of the cells' subviews have any constraints. Please add enough constraints to determine the location and size of all subviews.

For the cells with switches, you can actually use the `basic` cell type. Add the switch to the scene (NOT the view controller) and then hook it up as the accessory view of the cell:

![screen shot 2018-10-12 at 4 23 00 pm](https://user-images.githubusercontent.com/2257493/46898028-2db48e00-ce3b-11e8-8d80-dbdfe599948b.png)

That approach has a few advantages:

- No custom constraints needed.
- The cell looks like a standard cell (size, label position, etc.) because it is a standard cell.
- It Just Works with VoiceOver. Don't believe me? Try it!

Reply:
Thanks for the tip! I was able to convert all the cells to basic this way, not just the switches!